### PR TITLE
pull request

### PR DESCRIPTION
**Pull request by NescafeByBolyukh**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+.images-wrapper img {
+    margin: 10px;
+    height: 200px;
+    width: auto;
+}
+
+/*--------------3--------------*/
+.about-container {
+    height: 600px;
+    margin: 6px 0 12px 0;
```
Comment (karmeljuk) [inline css/style.css]: optimize margin

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+    grid-template-rows: 40% auto;
+    grid-row-gap: 6px;
+}
+
+.about-us {
+    background-color: #b0e0e6;
+}
+
+.what-we-do {
+    background-color: #ffff66;
```
Comment (karmeljuk) [inline css/style.css]: optimize color

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,230 @@
...
+}
+
+.third-article {
+    background-color: #ff80ed;
+}
+
+/*--------------5--------------*/
+.stories-container {
+    height: 400px;
+    margin: 10px 0 10px 0;
```
Comment (karmeljuk) [inline css/style.css]: optimize margin

File: css/style.css [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+    }
+
+    .article {
+        width: 49.8%;
+        height: 49.8%;
+    }
+
+    .first-article {
+        width: 100%;
+        margin-bottom: 0.4%;
```
Comment (karmeljuk) [inline css/style.css]: .4%

File: css/style.css [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+    }
+
+    .third-article {
+        margin-left: 0.4%;
```
Comment (karmeljuk) [inline css/style.css]: .4%

File: css/style.css [hunk 6]
```diff
@@ -0,0 +1,230 @@
...
+
+    .stories-container {
+        width: 59%;
+        height: 300px;
+    }
+
+    .subscribe-container {
+        width: 40.5%;
+        height: 200px;
+        margin-left: 0.5%
```
Comment (karmeljuk) [inline css/style.css]: .5%

**Review by karmeljuk — CHANGES_REQUESTED** [review]
Do minor css changes

**Comment by NescafeByBolyukh** [comment]
Done optimization of margins and bgcolor.

**Review by karmeljuk — APPROVED** [review]

**Comment by karmeljuk** [comment]
code ok

**Pull request merged**